beacon-chain/verification/data_column_test.go

```diff
@@ -0,0 +1,974 @@
+package verification
```

Comment:
```suggestion
package verification_test
```

Feel free to dismiss this request if it's not applicable or feasible. 

Could we switch this to a blackbox test and import the verification package? I think this is good practice and can help with some test dependency issues later down the road.

Reply:
Added a new item in the peerDAS TODO list.
Not doing it in this present PR, since it would impact a lot of non peerDAS specific (blob starting at Deneb) code as well.